examples/bevy_ggrs/src/main.rs

```diff
@@ -1,7 +1,8 @@
-use bevy::{log::LogPlugin, prelude::*, tasks::IoTaskPool};
+use bevy::{log::LogPlugin, prelude::*};
 use bevy_ggrs::{GGRSPlugin, Session};
+use bevy_matchbox::{MatchboxSocket, OpenSocketExt};
```

Comment:
We could do a bevy_matchbox::prelude to add all the commonly used types.

Reply:
I like this idea. Have added a prelude module with the types I think we need, a quick look at them would be much appreciated